web/src/components/workflow-graph/node-base.tsx

```diff
@@ -59,6 +66,23 @@ export default function NodeBase({
 							{type}
 						</Text>
 					</Flex>
+					{hasMissingSecret && (
+						<Flex
+							className="absolute top-2 right-2"
+							justify="end"
+							align="start"
+						>
+							<Tooltip content="Missing secret">
+								<Flex
+									className="bg-red-500 rounded-full w-6 h-6"
+									justify="center"
+									align="center"
+								>
+									<ExclamationTriangleIcon className="text-white w-4 h-4" />
+								</Flex>
+							</Tooltip>
+						</Flex>
+					)}
```

Comment:
Briefly tested it, this is also displayed for me if I click on "Create new workflow".